optax/_src/transform_test.py

```diff
@@ -202,6 +202,16 @@ def test_centralize(self, inputs, outputs):
     centralized_inputs, _ = centralizer.update(inputs, None)
     chex.assert_tree_all_close(centralized_inputs, outputs)
 
+  def test_sherman_morrisson(self):
```

Comment:
If we make the function `sherman_morrison` private, it shouldn't be tested here, but instead will be tested indirectly through the test of the public `scale_by_online_newton`.

Reply:
typo: extra s in morrison.